client/my-sites/earn/components/add-edit-plan-modal/index.tsx

```diff
@@ -228,7 +228,7 @@ const RecurringPaymentsPlanAddEditModal = ( {
 		ID: annualProduct?.ID, // will the the ID if already existing
 		price: currentAnnualPrice,
 		interval: PLAN_YEARLY_FREQUENCY,
-		title: productDetails.title + __( '(yearly)', 'jetpack' ),
+		title: `${ productDetails.title } ${ __( '(yearly)' ) }`,
```

Comment:
Do we need the "yearly" here btw?

In the list and in the popup we have other ways to show the "yearly" anyway, so it feels a bit repetative and extraneus:

<img width="433" alt="image" src="https://github.com/Automattic/wp-calypso/assets/87168/1941760c-cb9b-4bb3-8d35-06a60052018a">

<img width="513" alt="Screenshot 2023-11-10 at 15 13 51" src="https://github.com/Automattic/wp-calypso/assets/87168/2e852481-be14-4266-a330-cd3e8afbd911">